packages/core/src/indexing/postgres/init-tables.ts

```diff
@@ -110,6 +110,10 @@ export async function verifyTables(dataSource: Knex, modelsToIndex: Array<IndexM
 
   for (const tableName of tables) {
     const modelIndexArgs = modelsToIndex.find((model) => tableName == asTableName(model.model))
+    if (!modelIndexArgs) {
+      // FIXME: This means that there's is a table for a model that is no longer indexed. Should this table have been deleted?
```

Comment:
We originally said that expected behavior would be that the index data sticks around, but now i'm wondering if that's a good idea.  Or least, if we then decide to *re-index* the model, if we should delete and re-create the index table at that point?  Otherwise there could be very old outdated data in the index that we would start suddenly serving to the application.  Also would need to think about how this would work with historical indexing...

Reply:
I think we'll want to think about this, but we don't need to hold this PR up over it, can you file at ticket to revisit the behavior here after devcon?